docs/sources/userguide/usingdocker.md

```diff
@@ -68,13 +68,8 @@ You will see a list of all currently available commands.
 
 You can also zoom in and review the usage for specific Docker commands.
 
-Try typing Docker followed with a `[command]` to see the usage for that
-command:
-
-    $ docker attach
-    Help output . . .
-
-Or you can also pass the `--help` flag to the `docker` binary.
+Try typing a Docker command followed with flag `--help` to see the usage for
```

Comment:
Agreed, that's much better. I didn't want to diverge too much from the original text, but this is better :)